README.md

```diff
@@ -1,3 +1,141 @@
 # blinkmobile / angular-location [![npm](https://img.shields.io/npm/v/@blinkmobile/angular-location.svg?maxAge=2592000)](https://www.npmjs.com/package/@blinkmobile/angular-location) [![Travis CI Status](https://travis-ci.org/blinkmobile/angular-location.js.svg?branch=master)](https://travis-ci.org/blinkmobile/angular-location.js)
 
 Angular 1.x components for maps and geolocation
+
+
+## API
+
+
+### Configuration
+
+You must define a "googleMapsApiKey" property on the `$rootScope` with your API key.
```

Comment:
I'm going to postpone this part. I'm having trouble figuring out how to properly test AngularJS (1.x) components that have dependencies on constants. Either everything explodes, or no HTML is rendered. Quite frustrating.

I'd love to explore this separately riding shotgun with someone who as with more testing experience with AngularJS (1.x). :)